### DISCUSSION TITLE
Queue research by name (ordered research)

### DISCUSSION THREAD

**Discussion by Oinker** [Velociraptor115/DSPMods#18] [Ideas]
**Mod idea for somebody not me: Queue research by name (ordered research)**
Read text from the config file such as:
DSP Tech:
Electromagnetism
Automatic metallurgy
Basic logistics system
Basic assembling process
Mecha core Lvl 1
Electromagnetic matrix
Mass construction Lvl 1
Smelting purification
Drive engine Lvl 1
Thermal power
Energy circuit
Electromagnetic drive
Mass construction Lvl 2
Improved logistics system
Steel smelting
Environmental modification
Mechanical frame

**Comment by Velociraptor115** [comment]
I could implement the above, but with tech proto IDs
The ID numbers for each tech can be easily looked up from the Wiki
I can make them automatically cross-reference the English name for each tech, but the name could be changed due to a game update or translation patch, whereas the proto IDs are stable across updates

**Reply by Oinker** [reply]
It will be a big hit with r2modman and speedrunners
